client/src/constants/welcomeMessages.js

```diff
@@ -0,0 +1,14 @@
+export const DASHBOARD =
```

Comment:
Is it better to have these be constants or dynamic inputs to the Section component? If it was dynamic we could do some more personalized things with respect to messages.

Reply:
That is a good idea. But, the welcome messages will be retired.* [I have asked for direction.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1605888797021300)

If we were to use welcome messages long-term, then it would be better to have them be dynamic inputs. This could be achieved in two ways:
- __A.__ First, pass `welcomeMessageText` into `Section`, and then into `SectionMessages`. Next, set `welcomeMessageText` as child for `WelcomeMessage`.
- __B.__ First, pass `hasWelcomeMessage={false}` into `Section`, and then into `SectionMessages`. Next, (2) pass custom `Message` into `Section` (which is passed into `SectionMessages`).

_* Design begrudgingly agreed to welcome messages via [FP-327: […] Short-Term Solution](https://jira.tacc.utexas.edu/browse/FP-327), as a concession, in return for replacement by [FP-329: […] Long-Term Solution](https://jira.tacc.utexas.edu/browse/FP-329)._

Reply:
I am declining to perform this change, based on both [Tracy's answer](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1605889259021900?thread_ts=1605888797.021300&cid=CQUA4D5KJ) and because this change would further integrate a temporary—albeit well-written during its initial time crunch introduction—feature.

Reply:
@jchuahtacc Unresolved. Thoughts?